pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[project]
+name = "appgenerator"
+version = "1.0"
```

Comment:
ideally right now it's "0.20" though imho we don't really care much about having a version number but other contributors apparently insisted on it ... anyway ideally the python code app.py should fetch this value from pyproject.toml instead of the hardcoded value in app.py

Reply:
Yeah oops, i meant to remove this line actually :D